test_runner/performance/test_physical_replication.py

```diff
@@ -145,8 +147,12 @@ def test_replication_start_stop(
         conn_master = psycopg2.connect(master_connstr)
         cur_master = conn_master.cursor()
 
-        conn_replica = [psycopg2.connect(replica_connstr[i]) for i in range(num_replicas)]
-        cur_replica = [conn_replica[i].cursor() for i in range(num_replicas)]
+        conn_replica: List[Optional[psycopg2.connection]] = [psycopg2.connect(replica_connstr[i]) for i in range(num_replicas)]
```

Comment:
```suggestion
        conn_replica: List[psycopg2.connection] = [cast('psycopg2.connection', psycopg2.connect(replica_connstr[i])) for i in range(num_replicas)]
```

Reply:
What do you think of this?

Reply:
Sometimes I reset the elements to None, so I think it's correct for the type to be Optional